AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_materialproviders/smithygenerated/aws_cryptography_keystore/deserialize.py

```diff
@@ -0,0 +1,122 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
+
+import _dafny
+import aws_cryptographic_materialproviders.smithygenerated.aws_cryptography_keystore.dafny_to_smithy
+import module_
+from software_amazon_cryptography_keystore_internaldafny_types import (
+    CreateKeyOutput_CreateKeyOutput as DafnyCreateKeyOutput,
+    CreateKeyStoreOutput_CreateKeyStoreOutput as DafnyCreateKeyStoreOutput,
+    Error,
+    Error_KeyStoreException,
+    GetActiveBranchKeyOutput_GetActiveBranchKeyOutput as DafnyGetActiveBranchKeyOutput,
+    GetBeaconKeyOutput_GetBeaconKeyOutput as DafnyGetBeaconKeyOutput,
+    GetBranchKeyVersionOutput_GetBranchKeyVersionOutput as DafnyGetBranchKeyVersionOutput,
+    GetKeyStoreInfoOutput_GetKeyStoreInfoOutput as DafnyGetKeyStoreInfoOutput,
+    VersionKeyOutput_VersionKeyOutput as DafnyVersionKeyOutput,
+)
+from typing import Any
+
+from .dafny_protocol import DafnyResponse
+from .errors import (
+    CollectionOfErrors,
+    ComAmazonawsDynamodb,
+    ComAmazonawsKms,
+    KeyStoreException,
+    OpaqueError,
+    ServiceError,
+)
+from com_amazonaws_dynamodb.smithygenerated.com_amazonaws_dynamodb.shim import (
+    _sdk_error_to_dafny_error as com_amazonaws_dynamodb_sdk_error_to_dafny_error,
+)
+from com_amazonaws_kms.smithygenerated.com_amazonaws_kms.shim import (
+    _sdk_error_to_dafny_error as com_amazonaws_kms_sdk_error_to_dafny_error,
+)
+
+from .config import Config
+
+
+async def _deserialize_get_key_store_info(input: DafnyResponse, config: Config):
```

Comment:
I'm not certain actually... I wonder if this is used somewhere that I don't remember of the top of my head, or if this was used at some point but isn't used anymore...
I can look into this if it's blocking but would plan to leave it as-is otherwise